src/reconcile/rec_visibility.c

```diff
@@ -471,6 +474,7 @@ __wt_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, W
         first_upd = ins->upd;
     else {
         /* Note: ins is never null for columns. */
+        // WT-9417 IGNORE - Column store
```

Comment:
What does this mean?

Reply:
This ticket is a sort-of precursor to a future project that will review all asserts in WiredTiger, but we're making these changes now to help debugging of WT-9323. To keep the scope small and focused on WT-9323 I'm not touching any asserts that only apply to column store logic.

Effectively you can just ignore this comment. It's just to call out that there are asserts in src/reconcile that have been intentionally left unchanged.